qtism/data/storage/xml/XmlCompactDocument.php

```diff
@@ -313,9 +313,26 @@ public function schemaValidate($filename = '')
      */
     public function decorateRootElement(DOMElement $rootElement)
     {
-        $rootElement->setAttribute('xmlns', "http://www.imsglobal.org/xsd/imsqti_v2p1");
+        $namespaces = [
```

Comment:
What do u think about externalize this snippet to a Factory? Here u r relying of ``ttp://www.imsglobal.org/xsd/imsqti_%s` pattern, fingercrossed they will stick to it

Reply:
You're right, I'll change that, because it's too messy. I tried to make validation of the version, but the compact versions are just out of sync.
If you don't mind, Jérôme, I'll change qticompact 1.0 to 2.1 and 1.1 to 2.2, to align qticompact versions to qti versions, because validation is a nightmare.